src/IO/ADIOS/ADIOS2IOHandler.cpp

```diff
@@ -1527,10 +2101,12 @@ namespace detail
         finalized = true;
     }
 
-    void BufferedActions::configure_IO(ADIOS2IOHandlerImpl& impl){
+    void
+    BufferedActions::configure_IO( ADIOS2IOHandlerImpl & impl )
+    {
         ( void )impl;
         static std::set< std::string > streamingEngines = {
-            "sst", "insitumpi", "inline", "staging", "nullcore"
+            "sst", "insitumpi", "inline", "staging", "nullcore", "ssc"
```

Comment:
ssc slipped in here. we can generally leave it in or make it it's own PR (that would be cleaner)